.github/workflows/update-stars.yml

```diff
@@ -0,0 +1,35 @@
+name: Update stars count
+
+on:
+  schedule:
+    - cron: '0 0 * * 0' # Every Sunday at 00:00
```

Comment:
Do you think we'd get tired of having to merge a PR every week? I doubt things change that much. Maybe once a month would be enough?